atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -415,6 +416,12 @@ def adjust_build_kwargs(self):
             get_arrangement_version(self.workflow, self.build_kwargs['arrangement_version'])
 
     def get_platforms(self):
+        koji_platforms = self.workflow.prebuild_results.get(PLUGIN_CHECK_AND_SET_PLATFORMS_KEY)
+        if koji_platforms:
+            return koji_platforms
```

Comment:
Do we need to set `self.platforms`? Not sure if it's actually used somewhere else, but it does seem odd to have that variable pointing to incorrect data.

Reply:
That's how the original code had it, but I've gone ahead and changed it.

Reply:
Fixed.